src/apps/geoserver/webui/src/main/java/org/geoserver/cloud/autoconfigure/web/core/WebCoreConfiguration.java

```diff
@@ -29,6 +31,10 @@ public class WebCoreConfiguration {
         return new GeoServerWicketServlet();
     }
 
+    public @Bean TestWfsPost testWfsPostServlet() {
```

Comment:
for future reference, due to the issues with `TestWfsPost` upstream, we could wither propose a change or replace it completely with an amended version

Reply:
As part of future references.
Besides the semantics of TestWfsPost servlet is weird in the context of Geoserver (ie. the name is not clear compared with the use of it, etc), it would work fine if it's not because of these lines:

https://github.com/geoserver/geoserver/blob/7636c1dccc9331b8c6e8a7e660602b066b519001/src/wfs/src/main/java/org/vfny/geoserver/wfs/servlets/TestWfsPost.java#L348-L352

which are not allowing the servlet to be used in context of GSCloud.